convert2rhel/unit_tests/toolopts_test.py

```diff
@@ -145,3 +146,145 @@ def test_both_disable_submgr_and_no_rhsm_options_work(argv, raise_exception, no_
         convert2rhel.toolopts.CLI()
 
     assert convert2rhel.toolopts.tool_opts.no_rhsm == no_rhsm_value
+
+
+@pytest.mark.parametrize(
+    ("argv", "content", "output"),
+    (
+        (
+            mock_cli_arguments([""]),
+            "[subscription_manager]\npassword=conf_pass",
+            {"password": "conf_pass", "activation_key": None},
+        ),
+        (
+            mock_cli_arguments(["-p", "password"]),
+            "[subscription_manager]\nactivation_key=conf_key",
+            {"password": "password", "activation_key": None},
```

Comment:
For the cases where we are emitting a warning in the logs, should we also check that the log message shows up in caplog?

Reply:
Sure, good point :+1: